UPGRADE-3.2.md

```diff
@@ -74,8 +74,9 @@ Validator
 Yaml
 ----
 
- * Support for silently ignoring duplicate keys in YAML has been deprecated and
```

Comment:
BTW can't we add an example for this one? It looks hard to understand when you're not aware of the linked change.